docs/setup/production.asciidoc

```diff
@@ -134,6 +134,7 @@ Settings that must be the same:
 --------
 xpack.security.encryptionKey //decrypting session cookies
 xpack.reporting.encryptionKey //decrypting reports stored in Elasticsearch
+xpack.encryptedSavedObjects.encryptionKey // decrypting saved objects stored in Elasticsearch
```

Comment:
Not sure why, but "in Elasticsearch" doesn't show in the preview.

Reply:
@gchaps 
It does, code blocks are scrollable. It's not great though. Do you want to reformat using inline formatting?

Reply:
It would be better to show that the line is scrollable.  If you do inline formatting, make sure that it is consistent with the rest of the doc.

Reply:
> It would be better to show that the line is scrollable

That would be good. I don't know how to that. Can you help?